bedrock/base/templates/macros-protocol.html

```diff
@@ -246,20 +246,28 @@
   <a class="mzp-c-card-block-link" href="{{ link_url }}" data-link-name="{{ ga_title }}" data-link-type="link" data-link-group="card" {% if tag_label %}data-card-tag="{{ tag_label }}"{% endif %}>
     {% if image_url %}
     <div class="mzp-c-card-media-wrapper">
-      {{ image(
-        alt=image_alt,
-        class='mzp-c-card-image',
-        height=image_height,
-        highres_image_url=highres_image_url,
-        include_highres=include_highres_image,
-        include_l10n=l10n_image,
-        loading='lazy',
-        sizes=image_sizes,
-        sources=image_sources,
-        srcset=image_srcset,
+      {% set unclean = {
+        'alt': image_alt,
+        'class': 'mzp-c-card-image',
+        'width': image_width,
+        'height': image_height,
+        'loading': 'lazy',
+        'l10n': l10n_image
+      } %}
+      {% set optional_attributes = dict() %}
+      {% for key, value in unclean.items() %}
+        {% if value is not none and value is not false %}{{ optional_attributes.update({ key: value }) }}{% endif %}
+      {% endfor %}
+      {% if include_highres_image %}
+        {% set filename, ext = image_url.split('.') %}
+        {% set srcset = {"".join([filename, "-", "high-res", ".", ext]): "1.5x"} %}
```

Comment:
I'm not _in love_ with how I went about this, it just feels so messy compared to just passing the `resp_img` to the macro directly - which is how I did it in the Billboard PR #12299 

But happy with how I handled the `include_highres_image` attribute 😃

Reply:
I'm curious, why do we need to go this route with the card macro?

Reply:
Yeah, seeing the two in direct comparison feels like a clear win for simplicity of simply passing the image element needed.

In this case, it sounds like we would have a lot more instances of cards replace? We could split it out into separate tasks similar to the way we did the deprecated Protocol fixes last year (a spreadsheet with the card instances listed by url, and we just work through them as we find time)

Reply:
I took a bit of a closer look to dig deeper here, and one other thing we'd have to deal with is card data from www-admin and Contentful. But it seems like we could handle that OK and still simplify the `card` macro further (if we wanted to).

For example, we could construct a full formed image in the `content_card()` macro like so:

```jinja
{# Content Card: A card built from data in the database #}
{% macro content_card(name, content_cards=None) -%}
  {% set card_data = get_content_card(content_cards or page_content_cards, name) -%}
  {% if card_data -%}
    {% set image = '<img class="mzp-c-card-image" src="' ~ card_data.image_url ~ '" srcset="' ~ card_data.highres_image_url ~ ' 1.5x">' %}
    {% do card_data.pop('image_url') %}
    {% do card_data.pop('highres_image_url') %}
    {% do card_data.update({'image': image}) %}
    {{ card(**card_data) }}
  {% endif %}
{%- endmacro %}
```

Reply:
Yeah, I think going through and updating the calls to card is the better move here!

Reply:
I went this route only because there were so many calls to card that It would've been a bigger lift, but I think (as I noted above) the best move is to update each of the calls